app/src/androidTest/java/mozilla/lockbox/Navigator.kt

```diff
@@ -91,6 +91,9 @@ class Navigator {
 
     fun gotoFxALogin() {
         welcome { tapGetStarted() }
+        if (!FingerprintStore.shared.isDeviceSecure) {
```

Comment:
@sashei is it being used correctly here? At least all test are passing..but just to be sure...